department-of-veterans-affairs/va.gov-cms
# VACMS-5321: Set default value for vc intro text field.

## Description
See #5321

## Testing done
Visual / behat

## QA steps

- [] As an administrator, go to `admin/structure/types/manage/vet_center/fields/node.vet_center.field_intro_text` and visually verify the default value reads ```We offer confidential help for Veterans, service members, and their families at no cost in a non-medical setting. Our services include counseling for needs such as depression, posttraumatic stress disorder (PTSD), and the psychological effects of military sexual trauma (MST). We can also connect you with more support in VA and your community.```
- [ ] Spot check multiple vc nodes, and verify intro text matches the default.
- [ ] CHeck dblog to see if the update hook reported correctly and stopped running when it was supposed to.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
